style.css

```diff
@@ -0,0 +1,18 @@
+* {
+  text-align: center;
+  background-color: honeydew;
+}
+h2 {
+  font-family: Arial, Helvetica, sans-serif;
+  color: chartreuse;
+}
+li {
+  color: gray;
+  font-weight: bold;
+}
+ul {
+  display: inline-block;
+}
+ol {
+  display: inline-block;
+}
```

Comment:
A small trick to minimise repeated code is to use:

```css
ul, ol {
  display: inline-block;
}

```

Reply:
@Cerdtruir Forgot to mention that because all elements are centre aligned, the ordered and unordered lists look a bit wonkey. I recommend you correct that here.